ompi/mpi/man/man3/MPI_Fetch_and_op.md

```diff
@@ -0,0 +1,120 @@
+# Name
+
+`MPI_Fetch_and_op` - Combines the contents of the origin buffer with
+that of a target buffer and returns the target buffer value.
+
+# Syntax
+
+## C Syntax
+
+```c
+#include <mpi.h>
+
+int MPI_Fetch_and_op(const void *origin_addr, void *result_addr,
+
+    MPI_Datatype datatype, int target_rank, MPI_Aint target_disp,
+
+    MPI_Op op, MPI_Win win)
+```
+
+## Fortran Syntax (See Fortran 77 Notes)
+
+```fortran
+USE MPI
+! or the older form: INCLUDE 'mpif.h'
+
+MPI_FETCH_AND_OP(ORIGIN_ADDR, RESULT_ADDR, DATATYPE, TARGET_RANK,
+                 TARGET_DISP, OP, WIN, IERROR)
+    <type> ORIGIN_ADDR, RESULT_ADDR(*)
+
+    INTEGER(KIND=MPI_ADDRESS_KIND) TARGET_DISP
+    INTEGER DATATYPE, TARGET_RANK, OP, WIN, IERROR
+```
+
+## Fortran 2008 Syntax
+
+```fortran
+USE mpi_f08
+
+MPI_Fetch_and_op(origin_addr, result_addr, datatype, target_rank,
+        target_disp, op, win, ierror)
+    TYPE(*), DIMENSION(..), INTENT(IN) :: origin_addr
+    TYPE(*), DIMENSION(..) :: result_addr
+    TYPE(MPI_Datatype), INTENT(IN) :: datatype
+    INTEGER, INTENT(IN) :: target_rank
+    INTEGER(KIND=MPI_ADDRESS_KIND), INTENT(IN) :: target_disp
+    TYPE(MPI_Op), INTENET(IN) :: op
+    TYPE(MPI_Win), INTENT(IN) :: win
+    INTEGER, OPTIONAL, INTENT(OUT) :: ierror
+```
+
+
+# Input Parameters
+
+* `origin_addr` : Initial address of buffer (choice).
+* `result_addr` : Initial address of result buffer (choice).
+* `datatype` : Data type of the entry in origin, result, and target buffers
+(handle).
+* `target_rank` : Rank of target (nonnegative integer).
+* `target_disp` : Displacement from start of window to beginning of target buffer
+(nonnegative integer).
+* `op` : Reduce operation (handle).
+* `win` : Window object (handle).
+```
+
+
+# Output Parameter
+
+* `IERROR` : Fortran only: Error status (integer).
+
+# Description
+
+Accumulate one element of type `datatype` from the origin buffer
+`(origin_addr`) to the buffer at offset `target_disp`, in the target
+window specified by `target_rank` and win, using the operation op
+and return in the result buffer `result_addr` the contents of the target
+buffer before the accumulation.
+The origin and result buffers `(origin_addr` and `result_addr`) must be
```

Comment:
I think that these are supposed to be discrete paragraphs in the Description section.

I've seen this in a few pages now -- you might want to check and see if there's sorta-obvious-to-human line breaks (i.e., short line with an end of a sentence, and then a new line and a new sentence) in the rest of the sections in the rest of the pages.

You also might want to check your script and see if this is happening due to weird nroff source...?

Reply:
I think this happens when filtering all of the weird extra lines pandoc produces, I remember implementing some code to aggressively remove unwanted lines which is probably causing the issue